examples/node/spec/integration/apiIntegration.spec.ts

```diff
@@ -12,8 +12,8 @@ describe("Integration tests", () => {
   xit("should get an account", async () => {
     const params = {};
     const result = await client.api.v2010.accounts.page();
-    expect(result.end).toEqual(5);
```

Comment:
Either way is fine with me. The test doesn't currently run anyway since it's an `xit` test. Just commented out these lines since TS was complaining about them.